airflow/sentry.py

```diff
@@ -149,14 +149,21 @@ def add_breadcrumbs(self, task_instance, session=None):
 
         def enrich_errors(self, func):
             """Wrap TaskInstance._run_raw_task to support task specific tags and breadcrumbs."""
+            session_args_idx = find_session_idx(func)
```

Comment:
Is maybe need to change method annotation?
in airflow 2.0.0 this decorator wraps _run_mini_scheduler_on_child_tasks too. which is why the error actually occurred

Reply:
> which is why the error actually occurred

This is not the root cause. Just wrap `TaskInstance._run_raw_task` will also happen.